### DISCUSSION TITLE
SONOFF Zigbee 3.0 USB Dongle Plus mit redmatic und cuxd

### DISCUSSION THREAD

**Discussion by Derdernixweiss** [rdmtc/RedMatic#564] [Q&A]
Redmatic 7.2.1
cuxd 2.10
CCU3 Homematic
SONOFF Zigbee 3.0 USB Dongle Plus
Bei cuxd in setup folgende Zeile einfügen:
TTYASSIGN=ttyUSB0:NC

Wahrscheinlich ist der Post besser bei cuxd aufgehoben.

**Comment by hobbyquaker** [comment]
node-red-contrib-zigbee ist deprecated und auf RedMatic 9 nicht installierbar; Empfehlung ist zigbee2mqtt auf einem separaten Rechner (https://github.com/rdmtc/RedMatic/wiki/ZigBee). Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
